lib/spack/spack/package.py

```diff
@@ -56,7 +56,7 @@
 from llnl.util.tty.color import colorize
 from spack.filesystem_view import YamlFilesystemView
 from spack.util.executable import which
-from spack.stage import Stage, ResourceStage, StageComposite
+from spack.stage import _stage_prefix, Stage, ResourceStage, StageComposite
```

Comment:
Point 1:  Fair enough.
Point 2: Thanks.  I missed that in my notes.

Reply:
Done

Reply:
>     * I think it was also requested that the prefix of the stage dir name would be `spack-` rather than `spack-stage` (so a new variable might be needed)

@tgamblin, @scheibelp, @becker33 I'm not really comfortable with this change -- though I made it -- because this is used to determine which directories are purged by `spack clean`.  It seems more prudent to go back to the original idea of using (and purging) `spack-stage-*` instead of `spack-`, especially given `config:build_stage` customization incidents like #12633 .